packages/compass-saved-aggregations-queries/src/stores/index.ts

```diff
@@ -21,9 +30,24 @@ const _store = createStore(
     editItem: editItemReducer,
     appRegistry: appRegistryReducer,
   }),
-  applyMiddleware(thunk)
+  applyMiddleware(
+    thunk.withExtraArgument({
+      pipelineStorage,
+      queryStorage,
+    })
+  )
```

Comment:
Added storages as extra args in this store. It did trigger more changes in this plugin, but we have single instance of the class throughout and do not have to worry about `basepath` dependency for `FavoriteQueryStorage`